plugins/modules/hassio_core.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: hassio_host
+author: "marksie1988 / TotalDebug (@marksie1988)"
+short_description: Manage Home Assistant (HassIO) host
+version_added: "2.0.1"
+description:
+  - Manage Home Assistant (HassIO, hass.io) host - restart, update, stop
+options:
+  state:
+    description:
+      - State of host
+    required: true
+    choices: ['restarted', 'updated', 'stop']
+"""
+
+EXAMPLES = """
+# restart HassIO Core
+- hassio_host:
+    state: restarted
+"""
```

Comment:
documentation needs to be updated.

Reply:
I've updated documenation can you check if you agree?